foundation/foundation-mda/src/main/resources/profiles.json

```diff
@@ -2438,14 +2438,6 @@
 			}
 		]
 	},
-	{
-		"name": "aissemble-spark-worker-deploy",
-		"targetReferences": [
-			{
-				"name": "sparkWorkerJobFile"
```

Comment:
Q: Why does this need to be removed?

Reply:
Since the spark-worker docker image will be built by maven, we no longer need the Kubernetes job to build it in tilt for us.

Reply:
This is only used for spark-worker-docker Tilt docker-build so can be removed